testproject/Assets/ManualTests/ClientCounterBehaviour.cs

```diff
@@ -0,0 +1,616 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using MLAPI;
+using MLAPI.Messaging;
+
+/// <summary>
+/// This class is used to verify that the RPC Queue allows for the
+/// sending of ClientRpcs to specific clients.  It has several "direct"
+/// methods of sending as is defined in the enum ClientRpcDirectTestingModes:
+///     Single: This will send to a single client at a time
+///     Striped: This will send to alternating client pairs at a time
+///     Unified: This sends to all clients at the same time
+/// Each direct testing mode sends a count value starting at 0 up to 100.
+/// Each direct testing mode sends to all clients at least once during testing (sometimes twice in striped)
+/// During direct testing mode the following is also tested:
+///     all clients are updating the server with a continually growing counter
+///     the server is updating all clients with a global counter
+/// During all tests the following additional Rpc Tests are performed:
+///     Send client to server no parameters and then multiple parameters
+///     Send server to client no parameters and then multiple parameters
+/// </summary>
+public class ClientCounterBehaviour : NetworkBehaviour
+{
+    private const float  k_ProgressBarDivisor = 1.0f/200.0f;
```

Comment:
```suggestion
    private const float  k_ProgressBarDivider = 1.0f/200.0f;
```

divider vs divisor? 🤔

Reply:
Actually in this case it's neither, it's a scale-down factor ;)

Reply:
Actually, it is used as a divisor in the code base... it just handles dividing into zero without having to check.
**Divisor:**  a number by which another number is to be divided.